pages/shared/data/roadshow.yaml

```diff
@@ -16,70 +18,103 @@ nextstop:
 
 past:
   - title: NAIROBI
-    date: MAR 22, 2019  
+    coords: 65x33
+    date: MAR 22, 2019
   - title: LAGOS
-    date: MAR 19, 2019  
+    coords: 54x32
+    date: MAR 19, 2019
   - title: CAPE TOWN
-    date: MAR 15, 2019  
+    coords: 59x46
+    date: MAR 15, 2019
   - title: JOHANNESBURG
-    date: MAR 12, 2019  
+    coords: 63x43
+    date: MAR 12, 2019
   - title: HO CHI MINH CITY
+    coords: 88x24
     date: DEC 14, 2018
   - title: TORONTO
+    coords: 24x12
     date: AUG 30, 2017
   - title: SINGAPORE
+    coords: 86x29
     date: OCT 6, 2017
   - title: JAKARTA
+    coords: 89x32
     date: OCT 9, 2017
   - title: SYDNEY
+    coords: 103x44
     date: OCT 13, 2017
   - title: MUNICH
+    coords: 56x12
     date: NOV 7, 2017
   - title: PARIS
+    coords: 54x14
     date: NOV 10, 2017
   - title: MADRID
+    coords: 51x17
     date: NOV 14, 2017
   - title: LONDON
+    coords: 50x12
     date: NOV 16, 2017
   - title: BEIJING
+    coords: 90x19
     date: DEC 5, 2017
   - title: SHANGHAI
+    coords: 90x21
     date: DEC 8, 2017
   - title: BUENOS AIRES
+    coords: 24x42
     date: MAR 13, 2018
   - title: BOGOTA
+    coords: 20x28
     date: MAR 16, 2018
   - title: MEXICO CITY
+    coords: 15x22
     date: MAR 21, 2018
   - title: CHICAGO
+    coords: 20x13
     date: MAR 27, 2018
   - title: SUNNYVALE
+    coords: 8x16
     date: APR 4, 2018
   - title: TAIPEI
+    coords: 92x21
```

Comment:
Taipei's in the water now. Can we get it on to one of the little cute blue stylized islands?

Reply:
I moved Taipei down to the next island, but it seems it is in Indonesia now and not Taiwan. Maybe we should just update the cities-map.svg itself and add more island?